src/test/java/integration/Nextflow.java

```diff
@@ -56,7 +56,15 @@ void nextflowFromGLSTutorial() throws IOException {
     assertEquals(0, exitCode, "workspace created without errors");
 
     // run the script that downloads the NF workflow from GH and runs it
-    exitCode = TestBashScript.runScript("NextflowRnaseq.sh");
+    TestBashScript.runScript("NextflowRnaseqSetup.sh");
+
+    // Try running the script once, which will likely (but not definitely) fail due to missing
+    // permissions on the bucket.
+    TestBashScript.runScript("NextflowRnaseqWorkflow.sh");
```

Comment:
Would it help to retry only if the first run failed? i.e. check the exit code and retry if needed. 
A retry on success wouldn't cause a problem here since Idempotence is not required for this script

Reply:
Good call, that potentially saves 10-15 minutes each run